src/components/SecuritySection.js

```diff
@@ -38,9 +38,11 @@ const useStyles = makeStyles((theme) => ({
   },
   subtitle: {
     textAlign: 'center',
+    width: '90%',
```

Comment:
Looks like the second paragraph is still wide, wider than the first paragraph?
https://www.chromatic.com/pullrequest?activeElementId=comparison-test-key-618b0e435e3b69e68a2c3429-1200&appId=6154d1bab7cd94003aee223d&number=230&view=changes